### ISSUE TITLE
No power management | CFGLocked

### ISSUE THREAD

**Issue by Dakros66** [AniKulkarn/Hackintosh-ThinkPad-E14#27]
There is bad power management, cfg is locked at bios so no correct power of CPU is applicated.

**Comment by Dakros66** [comment]
I add that i have ~60℃ In idle, is that ok? Or you have better temps?

**Comment by AniKulkarn** [comment]
It has to do with the CPUFriend.kext configuration. While setting up your Hackintosh, you must have used CPUFriendFriend as a tool to configure the kext, right? What power profile did you choose there?

**Comment by Dakros66** [comment]
Yes, i have been testing and i have something bad i think, with my own CPUfriend.kext (balanced) i see that the fan is not working, but if i inject your .kext (main tree) the fan runs ok with temperatures about 50-55 to 70-80 peaks, only runing sublime text and 2 or 3 Chrome tabs.

Battery 100 to 5% take about 2:30/3h only wifi, 65% brightness and chrome/ terminal/ sublime text.

In your main branch the CPUfriend.kext what mode is using?

Any idea about no fan problem with my own kext?


Regards.

**Comment by AniKulkarn** [comment]
I sort of don't remember the profile I used haha it was a long time ago. But I reckon it was balanced too! 
Btw, Chrome makes my computer drain really really fast. I stopped using Chrome on my Hackintosh. Switched to Safari. 

I'm not sure why your kext is making your computer overheat even on the balanced mode. I would recommend a trial and error approach here :D. Just try all profiles and see which one helps you! If none do, just use my kext LOL.

**Comment by Zer0xPoint** [comment]
is there any way to unlock cfg?

**Comment by AniKulkarn** [comment]
@Zer0xPoint not sure. I've been on a huge break for Hackintoshing due to my workload!
